lms/djangoapps/teams/static/teams/js/views/edit_team_members.js

```diff
@@ -22,6 +22,7 @@
                 },
 
                 initialize: function(options) {
+                    this.options = _.extend({}, options);
```

Comment:
I think this could be better written as
```javascript
this.options = _.extend({}, this.options, options);
```
this will make it future proof if options added to this view as well.